### DISCUSSION TITLE
opset 14 support

### DISCUSSION THREAD

**Discussion by LeoAtGit** [microsoft/onnxruntime#7450] [Other Q&A] [answered]
Hi,
I wanted to ask if ONNX opset 14 is already supported in the onnxruntime? The only relevant documentation I could find was in /docs/Versioning.md, but unfortunately it is already outdated.

If opset 14 is not currently supported, how long does it usually take for a new opset to get implemented?

On a final note, does anyone know how long it takes for the packages to get updated in NuGet?

Thanks!

**Comment by pranavsharma** [comment, marked as answer]
opset14 will be supported in the upcoming ORT 1.8 release with nuget pkgs available around 5/25 (tentatively).

**Reply by LeoAtGit** [reply]
Thank you for the fast answer
